### PR TITLE
 added is_active flag to profile serializer

### LINKED ISSUE #404: Add is_active flag to profile serializer
Let's return the `is_active` flag in all the profile endpoints for `v2` of the API.

Use Case: https://github.com/mozilla/network-pulse/issues/980#issuecomment-413727349

### PR DESCRIPTION
**Pull request by ali-alsabbah**
Fix #404

### CODE REVIEW

File: pulseapi/profiles/serializers.py [hunk 1]
```diff
@@ -96,6 +96,11 @@ class UserProfileSerializer(serializers.ModelSerializer):
         allow_blank=True,
     )
 
+    is_active = serializers.BooleanField(
+        required=False,
```
Comment (gideonthomas) [inline pulseapi/profiles/serializers.py]: let's make this read_only as well.

Comment (gideonthomas) [reply pulseapi/profiles/serializers.py]: you also need to remove this from the `exclude` option in the `Meta` class

File: pulseapi/profiles/serializers.py [hunk 2]
```diff
@@ -162,6 +167,7 @@ def to_representation(self, obj):
         return {
             'id': obj.id,
             'name': obj.name
```
Comment (gideonthomas) [inline pulseapi/profiles/serializers.py]: always end every pair in a dict with a comma

File: pulseapi/profiles/serializers.py (hunk 1 above)
Comment (ali-alsabbah) [reply pulseapi/profiles/serializers.py]: done

File: pulseapi/profiles/serializers.py (hunk 2 above)
Comment (ali-alsabbah) [reply pulseapi/profiles/serializers.py]: done

File: pulseapi/profiles/serializers.py [hunk 3]
```diff
@@ -96,6 +96,11 @@ class UserProfileSerializer(serializers.ModelSerializer):
         allow_blank=True,
     )
 
+    is_active = serializers.BooleanField(
+        required=False,
+        allow_blank=True,
```
Comment (gideonthomas) [inline pulseapi/profiles/serializers.py]: remove this since it's not a valid option for `BooleanField`

Comment (gideonthomas) [reply pulseapi/profiles/serializers.py]: yeah probably...I don't think it needs the `required` option since it will always be there.

Comment (ali-alsabbah) [reply pulseapi/profiles/serializers.py]: done

File: pulseapi/profiles/serializers.py [hunk 4]
```diff
@@ -96,6 +96,9 @@ class UserProfileSerializer(serializers.ModelSerializer):
         allow_blank=True,
     )
 
+    is_active = serializers.BooleanField(
+    )
```
Comment (gideonthomas) [inline pulseapi/profiles/serializers.py]: this shouldn't be on a new line if it isn't taking any parameters.

Comment (ali-alsabbah) [reply pulseapi/profiles/serializers.py]: fixed

**Comment by gideonthomas** [comment]
Thanks @ali-alsabbah!

**Pull request merged**